.gitignore

```diff
@@ -30,5 +31,9 @@ README-svn.txt
 !/source/wp-content/themes
 !/source/wp-content/themes/wporg-news-2021
 
+# And this plugin.
+!/source/wp-content/plugins
+!/source/wp-content/plugins/wporg-post-title
```

Comment:
🤔 , I'd lean towards putting the code in the `blocks/` folder in the `mu-plugins` repo, so that all sites can use it. I'd guess that future redesigns will have a similar concept, with part of the post title being used as a design element.

I don't feel strongly, though.

Reply:
I don't think other sites will need this exact functionality, but if we leave it here that's a third folder we would need to sync into SVN.

Reply:
If I change it to a more specific block like post-title-version, would any other sites have a use for it? It's a pretty obscure use case. Agree that mu-plugins could make sense for a more generally reusable block though.

Reply:
Kelly has a good point. I could even seen this being put in `functions.php` since it's really just a glorified custom template tag. I wouldn't do that for normal blocks, but this is just a workaround for FSE limitations. I don't feel strongly though.

Reply:
I'm okay with leaving it in the theme (maybe a `blocks` folder, in case we need any other workarounds). If we're still going the parent-theme route, they'll be available to any child themes too. I do think we should use a `block.json` + set it up in JS too, so that it won't break in the site editor (see https://github.com/WordPress/wporg-mu-plugins/pull/26) when we get there.